src/libs/base64.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @url http://jsperf.com/blob-base64-conversion
+ */
+export default class Base64 {
+	static blobToBase64(blob) {
```

Comment:
Maybe we can rewrite this lib in promises? Should we move this code to a place where it used so we could exclude all libs from /src/libs at the end?